### PR TITLE
Pre-import demo data in the demo node

### PR DESCRIPTION
**Pull request by tobim**
This moves the demo data import from the container startup to the image creation phase.
I also added 2 alternative stopped pipelines that are continuous by nature.

### CODE REVIEW

File: demo-node/entrypoint-noingest.bash [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# of Tenzir processes.
+export TENZIR_CONSOLE_VERBOSITY="${TENZIR_CLIENT_CONSOLE_VERBOSITY:-"info"}"
+
+# Wait until the HTTP endpoint is listening.
+while ! timeout 100 bash -c "echo > /dev/tcp/127.0.0.1/5160"; do
+  echo "Waiting for the HTTP service..."
+  sleep 1
+done
+
+nic_pipe="from nic eth0 | drop data | import"
```
Comment (mavam) [inline demo-node/entrypoint-noingest.bash:23]: Where do we provide capture capabilities to the container?

**Review by mavam — APPROVED** [review]
Approving the code, but I haven't tested it.

File: demo-node/entrypoint-noingest.bash (hunk 1 above)
Comment (tobim) [reply demo-node/entrypoint-noingest.bash:23]: There is no need to. `pcap.packet` data does show up in the database, unfortunately at a very low rate, our app traffic seems to take some other path. Unfortunately `shell "ip -j a"` does not work in fargate.

File: demo-node/import.bash [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+trap 'trap " " SIGTERM; kill 0; wait' SIGINT SIGTERM
+
+coproc NODE { exec tenzir-node --print-endpoint; }
+# shellcheck disable=SC2034
+read -r -u "${NODE[0]}" DUMMY
+
+tenzir 'read suricata | where #schema != "suricata.stats" | import' < Suricata/eve.json
+cat Zeek/*.log | tenzir 'read zeek-tsv | import'
+rm -rf Suricata
+rm -rf Zeek
```
[the remark this answers is not in the document]
Comment (tobim) [reply demo-node/import.bash]: The image size seems to be almost unaffected, so it seems fine to leave them.

**Pull request merged**